app/src/components/ScrabbleBoard.js

```diff
@@ -1,13 +1,45 @@
 import './ScrabbleBoard.scss'
 import ScrabbleTile from './ScrabbleTile'
+import { useContext } from 'react';
+import { GameContext } from '../contexts/GameContext';
+import { HandContext } from '../contexts/HandContext';
+import { currentBoardMapper } from '../functions';
+
+export default function ScrabbleBoard({ currentBoard }) {
+
+    const { currentTile, addToCurrentMove, currentMove, setCurrentTile, setCurrentBoard } = useContext(GameContext);
+    const { currentHand, setCurrentHand } = useContext(HandContext);
+
+    async function onTileClick(value, score, id, rowIndex, columnIndex) {
+        if (!value && !score && !id && currentTile) {
+            addToCurrentMove({ x: rowIndex, y: columnIndex, tile: currentTile });
+            setCurrentHand(currentHand.filter(tile => tile.id !== currentTile.id));
+            const newBoard = await currentBoardMapper(currentBoard, rowIndex, columnIndex, currentTile);
```

Comment:
Ezt itt ne csináld!! Az új boardot csak akkor fogjuk setelni, hogyha a felhasználó megnyomja a gombot, hogy "Lerak". Egyelőre csak a hand-ben tárold el ezeket, és a `ScrabbleBoard` komponensnek gondoskodnia kell róla, hogy ezek is ki legyenek rajzolva, csak más style-al (úgy látom ezt már meg is csináltad)